tools/api-markdown-documenter/package.json

```diff
@@ -24,10 +24,11 @@
 	},
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"build": "concurrently npm:build:compile npm:lint && npm run build:readme && npm run build:docs",
+		"build": "concurrently npm:build:compile npm:lint && npm run build:examples && npm run build:readme && npm run build:docs",
```

Comment:
Yes - we need to build the examples _after_ we build the code, where it was built simultaneously before.

Reply:
Oh, and is that because the examples try to import from the package itself? If they were importing from some external dependency we would be able to build source and examples in parallel? I'm fine with this approach, just wanted to understand the reason.

Reply:
Yeah, because we're importing from the package itself.